packages/cache-utils/tests/helpers/main.js

```diff
@@ -18,14 +18,29 @@ const createTmpDir = async function (opts) {
   return path
 }
 
-const createTmpFile = async function (opts) {
+const createTmpFile = async function ({ name, ...opts } = {}) {
   const tmpDir = await createTmpDir(opts)
-  const filename = basename(await tmpName())
+  const filename = name || basename(await tmpName())
   const tmpFile = join(tmpDir, filename)
   await pWriteFile(tmpFile, '')
   return [tmpFile, tmpDir]
```

Comment:
Could this function re-use `createTmpFiles()` internally (to avoid code-duplication)?

Reply:
Good call 👍 fixed in https://github.com/netlify/build/pull/3516/commits/47325fd48e55136e623987be4662a474a527d3c7